components/layout/index.jsx

```diff
@@ -1,20 +1,36 @@
-import React from 'react'
+import React, { useState } from 'react'
+import classNames from 'classnames'
 import Header from '../header'
 import Head from '../head'
 import './index.css'
 import Sidebar from '../sidebar'
 
 const Layout = ({ children }) => {
+  const [isSidebarVisible, toggleSidebarVisibility] = useState(false)
+
+  const handleCLick = e => {
+    const { name, value } = e.target
```

Comment:
`e` may be confusing, I might mean either `event` or `error`, and both are quite common everywhere. I would prefer call `event` actually `event`.

Reply:
ok fair enough